frontend/ui/widget/notification.lua

```diff
@@ -28,6 +32,15 @@ local Notification = InputContainer:new{
     toast = true, -- closed on any event, and let the event propagate to next top widget
 
     _nums_shown = {}, -- array of stacked notifications
+
+    SOURCE_NONE = 0,
+    SOURCE_BOTTOM_MENU_ICON = 1,
+    SOURCE_BOTTOM_MENU_TOGGLE = 2,
```

Comment:
We need this distinction ?

Reply:
Not necessarily. But it does no harm. 
Maybe some popup messages of icons are usefull in koptoptions (if the user does not realize what an icon does then the popup can tell this.) -> if we don't like them at the end we can squash those two together.